server/templates/base.html.j2

```diff
@@ -11,39 +11,85 @@
+              </span>
+            </a>
+          {% elif offering %}
+            <a href="{{ url_for('offering', offering=offering) }}">
+              <span class="hidden-small mdl-layout__title">
+                {{ offering.name }}
+              </span>
             </a>
-            <ul class="mdl-menu mdl-menu--bottom-left mdl-js-menu" for="room-menu">
-              <a class="mdl-menu__item add-item" href="{{ url_for('new_room', exam=exam) }}"><i class="material-icons">add</i>
-                <span>Add rooms...</span>
+          {% else %}
+            <a href="{{ url_for('index') }}">
+              <span class="hidden-small mdl-layout__title">
+                {{ title }}
+              </span>
+            </a>
+          {% endif %}
+          
+          <div class="hidden-small mdl-layout-spacer"></div>
+
+          {% if exam %}
+            <nav class="mdl-navigation">
+              <a id="room-menu" class="mdl-navigation__link" href="#">
+                Rooms <i class="material-icons">expand_more</i>
+              </a>
+              <ul class="mdl-menu mdl-menu--bottom-left mdl-js-menu" for="room-menu">
+                <a class="mdl-menu__item add-item" href="{{ url_for('import_room', exam=exam) }}"><i class="material-icons">add</i>
+                  <span>Add rooms...</span>
+                </a>
+              {% for room in exam.rooms %}
+              <a href="{{ url_for('room', exam=exam, name=room.name) }}">
+                <li class="mdl-menu__item">
+                    {{ room.display_name }}
+                </li>
               </a>
-            {% for room in exam.rooms %}
-            <a href="{{ url_for('room', exam=exam, name=room.name) }}">
-              <li class="mdl-menu__item">
-                  {{ room.display_name }}
-              </li>
+              {% endfor %}
+              </ul>
+              <a class="mdl-navigation__link" href="{{ url_for('students', exam=exam) }}">Students</a>
+              <a class="mdl-navigation__link" href="{{ url_for('offering', offering=exam.offering) }}">All Exams</a>
+              <a class="mdl-navigation__link" href="{{ url_for('index') }}">All Offerings</a>
+              <a class="mdl-navigation__link" href="{{ url_for('help') }}">Help</a>
+            </nav>
+          {% elif offering %}
+            <nav class="mdl-navigation">
+                <a class="mdl-navigation__link" href="{{ url_for('index') }}">All Offerings</a>
+                <a class="mdl-navigation__link" href="{{ url_for('help') }}">Help</a>
+            </nav>
+          {% else %}
+            <nav class="mdl-navigation">
+                <a class="mdl-navigation__link" href="{{ url_for('index') }}">All Offerings</a>
+                <a class="mdl-navigation__link" href="{{ url_for('help') }}">Help</a>
+            </nav>
+          {% endif %}
+          {% if current_user.is_authenticated %}
+            <a id="auth-menu" class="mdl-navigation__link" href="#">
```

Comment:
same here, needs to be a button. BS5 docs show a good example of the complete aria attributes. Does flask have view helpers we could write?